tests/acceptance/features/apiGraph/unassignRole.feature

```diff
@@ -0,0 +1,96 @@
+@api
+Feature: delete role
+  As an admin
+  I want to delete roles of users
+  So that the role of user is set to default
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  @issue-5938
+  Scenario: admin user unassign the role of another user with admin role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Admin" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    # issue-5938 is related to below step, remove the comment when the issue is fixed
+    And user "Brian" should not be able to create a group "mygroup"
+    And user "Brian" should have the role "User" assigned
+
+  @issue-5938
+  Scenario: admin user unassign the role of another user with space admin role using graph API
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Admin" to user "Alice" using the Graph API
+    And the administrator has assigned the role "Space Admin" to user "Brian" using the Graph API
+    When the user "Alice" unassigns the role of user "Brian" using the Graph API
+    Then the HTTP status code should be "204"
+    And user "Brian" should not have any role assigned
+    # issue-5938 is related to below step, remove the comment when the issue is fixed
+    And user "Brian" should not be able to create a space "Project Mars" of type "project"
```

Comment:
we check it here: https://github.com/owncloud/ocis/blob/2caf88c18bcb7b78b4f8b82518a384317907fda3/tests/acceptance/features/apiSpaces/createSpace.feature#L11
please delete this step. make this text work. Goal of this test: check that Brian has User role of the end